src/layouts/EditHomepage/HomepagePreview.tsx

```diff
@@ -200,12 +201,16 @@ export const HomepagePreview = ({
           {/* TODO: Infocols section */}
           {EditorHomepageFrontmatterSection.isInfocols(section) && (
             <>
-              <div
+              <TemplateInfocolsSection
                 key={`section-${sectionIndex}`}
+                title={section.infocols.title}
+                subtitle={section.infocols.subtitle}
+                linktext={section.infocols.linktext}
+                url={section.infocols.url}
+                infoboxes={section.infocols.infoboxes}
```

Comment:
not a blocker, wouldn't using a spread over `section.infocols` be better here?

Reply:
true, was following the same format as the above components. updated now

Reply:
done